src/containers/wallet/Wallet.tsx

```diff
@@ -112,6 +121,13 @@ class Wallet extends React.Component<DecoratedProps, State> {
   }
 
   componentWillMount() {
+    // store info if the current platform is supported
+    if (typeof inobounce.supported === 'undefined') {
+      inobounce.supported = inobounce.isEnabled();
+    }
+    if (inobounce.supported) {
+      inobounce.enable();
```

Comment:
From iNoBounce README/docs:

> Enable iNoBounce. It's enabled by default on platforms that support -webkit-overflow-scrolling, so you only need to call this method if you explicitly disable it or want to enable it on a platform that doesn't support -webkit-overflow-scrolling.

If the above is to be believed the code blocks in componentWillMount and componentWillUnmount seem redundant OR there's a bug here that the iNoBounce is enabled for non-Wallet components as well by default.

Reply:
The key phrase here is:
> or want to enable it on a platform that doesn't support -webkit-overflow-scrolling

Once disabled, its enabled for every platform, thats why the result of autodetection is stored before calling `enable`.

I'll move `inobounce.supported = inobounce.isEnabled();` to `App.tsx` so it'll looks more global.